QuickStartGuide.md

```diff
@@ -0,0 +1,134 @@
+## Introduction
+
+GitHub Actions can be run in GitHub-hosted cloud or self hosted environments. Self-hosted runners offer more control of hardware, operating system, and software tools than GitHub-hosted runners provide
+
+With just a few steps, you can set up your kubernetes (K8s) cluster to be a self-hosted environment.
+In this guide, we will setup pre-requistes ,deploy Actions Runner controller (ARC) and then target that cluster to run GitHub Action workflows.
```

Comment:
Typo 
prerequisite, deploy

Reply:
Thanks for the comment, fixed in the latest commit.